multigeiger/multigeiger.ino

```diff
@@ -506,7 +506,6 @@ void loop()
   update_display = (GMC_counts >= MAXCOUNTS) || ((current_ms - time2display) >= DISPLAYREFRESH);
   if(update_display) isr_GMC_counts = 0;
   count_timestamp = isr_count_timestamp;
-  // GMC_counts_2send = isr_GMC_counts_2send;
   count_timestamp_2send = isr_count_timestamp_2send;
```

Comment:
if the first line is not needed (which seems true), isn't the 2nd one not needed either?

Reply:
(and still, these cleanups do not belong into this PR, see topic)

Reply:
ping

Reply:
ok, PR is out.